examples/01a-Getting-started.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>It seems that the dataloader does not support cat_names, cont_names and label_names - is that correct?</p><p>It seems it relies on a data schema -&gt; Either, we need to provide an easy tool to define a schema manually or add the 3 parameters back?</p><p> </p><p>Can we make this more generic/explanable. E.g.</p><p> </p><p>label_columns = ['rating']</p><p>def process_batch(data, _):</p><p>x = {col: data[col] for col in data.keys() if col not in label_columns}</p><p>y = data[label_columns]</p><p>&nbsp;&nbsp;return (x, y)</p><p> </p><p>"What <code>Tensorflow</code> expects to see are targets as the 2nd position in the tuple. " -&gt; Let's rewrite it to make it more clear - something like</p><p>"TensorFlow Kera's .fit function expects the data to get as a tuple (x, y), with x are the input features and y is the label. We need to provide this information to the dataloader. We can add a custom function to convert the data into the tuple with <code>process_batch</code>"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>this reads much better now and communicates much more to the reader 🙂👍</p>